TweetMate/src/main/java/com/ybsystem/tweetmate/resources/ResColor.java

```diff
@@ -0,0 +1,88 @@
+package com.ybsystem.tweetmate.resources;
+
+import android.util.TypedValue;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.databases.PrefWallpaper;
+
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class ResColor {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1118)